docs/sources/operators-guide/architecture/components/ruler/index.md

```diff
@@ -14,6 +14,17 @@ Each tenant has a set of recording and alerting rules and can group those rules
 
 ![Architecture of Grafana Mimir's ruler component](ruler.svg)
 
+## Operational modes
+
+The ruler supports two different rule evaluation modes, adaptable according to the specific needs of the user:
+
+- **Internal:** This is the default mode. When set, the ruler connects directly through its built-in querier to ingesters and store-gateways, and internally evaluates the rule expression to generate the resulting series.
```

Comment:
Instead of a list, I would convert it to sub-paragrapths (e.g. `### Internal`). This way we can move the architecture diagram here.

Reply:
I would add a noun after this adjective. Internal [what?]

Reply:
Given that there will be few if any links to the specific modes, let's keep the section as is and we can still add the image(s).

Reply:
The problem is that the diagram above refers only to the "Internal" mode and if we keep it above looks like how the ruler works in any operational mode, which is wrong.